### DISCUSSION TITLE
Autobed level gets to Probing Point 157/225

### DISCUSSION THREAD

**Discussion by apr1967** [Jyers/Marlin#632] [General Discussion]
Installed E3V2-UBL-BLTouch-15x15-HS-v4.2.2.

Did a manual level using the firmware (nice!)
Set Z
Turned on UBL and had it probe.  It gets to point 157/225 and stops (no error) saying you must remember to save it as its only in ram.

Is 157 points all it is supposed to probe?  15x15 = 225 but I didn't learn new math, so I'm a bit confused.

Nope 157 is correct.  Not all points are probe able.  Found that by searching on 157.

Am I following the right steps?  After this is done, I would assume I am ready to do a test print, or should I check X again?

Because my simple test print is still failing.

As long as my mesh is saved in ram I should just be able to run any gcode, a home will automatically reload the mesh correct, or do I need to add the m240 -L?

**Comment by Jyers** [comment]
See #419 and #403. I'm working on a status message will hopefully make this more clear in the future. As far as the gcode take a look here: https://github.com/Jyers/Marlin/wiki/Configuration-Details The UBL section details exactly what is needed for UBL to function properly.

**Reply by apr1967** [reply]
Thanks.
